test/common/runtime/BUILD

```diff
@@ -31,6 +42,18 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_test(
+    name = "runtime_flag_override_test",
+    srcs = ["runtime_flag_override_test.cc"],
+    args = [
+        "--runtime-feature-override-for-tests=envoy.reloadable_features.test_feature_false",
```

Comment:
FWIW, this seems like a generally useful feature for runtime configuration where people don't want to use the filesystem (CLI/gflags implementation like we discussed). Is it worth adding a TODO or opening a help wanted issue on that?

Reply:
Also, if possible can you rename `envoy.reloadable_features.test_feature_false` to something else? I found it pretty confusing to try to sort out the logic with the built in `envoy.reloadable_features.test_feature_true`. Maybe more comments would help also or just a different name like `envoy.reloadable_features.injected_test_feature` and `envoy.reloadable_features.built_in_test_feature` or something?

Reply:
Do you think other folks would use this?  I thought from prior discussions you thought google was flag happy but most folks wanted to use the existing runtime.
I think if we're going to use it out of test we have to allow for feature=true,feature2=false which I can do pretty easily.

Not sure if removing test_feature_false helps - I've clarified the language in both tests to hopefully clarify things.

Reply:
I do think that Google is an outlier in using flags for this type of stuff, but I was just suggesting that this code might be generally useful to someone, so probably worth tracking somewhere in case it comes up again. I think it's totally fine to leave as test only code for now.

Thanks for all the comments, it's much easier to understand for me now.